tests/scripts/run-throughput-perf.py

```diff
@@ -417,6 +427,9 @@ def main(args):
                 "--config",
                 "OptLevel",
                 opt_level,
+                "--config",
```

Comment:
As Jarret mentions, we used to have jobs for jit32, which we listed as a different architecture to match outerloop tp jobs (it was x86jit32). So we may want to copy that behavior here.